package.json

```diff
@@ -22,32 +22,30 @@
   },
   "homepage": "https://github.com/MarlaJane/witch-guide-api#readme",
   "dependencies": {
-    "body-parser": "^1.19.0",
+    "axios": "^0.19.2",
     "express": "^4.17.1",
     "mysql2": "^2.1.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-router-dom": "^5.2.0",
     "sequelize": "^5.21.10",
     "sequelize-cli": "^5.5.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.2",
+    "@babel/core": "^7.10.3",
     "@babel/preset-env": "^7.10.2",
     "@babel/preset-react": "^7.10.1",
     "babel-loader": "^8.1.0",
     "babel-watch": "^7.0.0",
     "chai": "^4.2.0",
     "concurrently": "^5.2.0",
     "env-cmd": "^10.1.0",
-    "eslint": "^7.0.0",
-    "eslint-config-airbnb": "^18.1.0",
+    "eslint": "^7.3.1",
```

Comment:
You never updated your lint scripts to scan your JSX files

![image](https://user-images.githubusercontent.com/5775716/86019153-417b9b80-b9f4-11ea-862a-0767aa498669.png)